packages/lib/event-types/transformers/api-request.ts

```diff
@@ -102,6 +117,51 @@ function transformApiEventTypeBookingFields(
   return customBookingFields;
 }
 
+function transformApiEventTypeIntervalLimits(
```

Comment:
Would be good to add tests of functions added to `api-request.ts` in `api-request.spec.ts` - it should be fast to add and would be good because transformation logic can sometimes be hard to grasp right away.

Reply:
added https://github.com/calcom/cal.com/pull/16128/commits/b8bf222e894147dc9d0fc055443188cecc25cf29

Reply:
Great!